lib/libutee/include/tee_arith_internal.h

```diff
@@ -73,7 +73,7 @@
  * It defines the size of the scratch memory pool for the underlying
  * mpa library.
  */
-#define TEE_MAX_NUMBER_OF_SUPPORTED_BITS 2048
+#define TEE_MAX_NUMBER_OF_SUPPORTED_BITS	CFG_TA_BIGNUM_MAX_BITS
```

Comment:
Here too I think we should not bother with two names for the same thing =>  `s/TEE_MAX_NUMBER_OF_SUPPORTED_BITS/CFG_TA_BIGNUM_MAX_BITS/`.